robot/client/client.go

```diff
@@ -1036,6 +1063,12 @@ func (rc *RobotClient) TransformPointCloud(ctx context.Context, srcpc pointcloud
 	return output, nil
 }
 
+// CurrentInputs returns a map of the current inputs for each component of a machine's frame system
+// and a map of statuses indicating which of the machine's components may be actuated through input values.
+func (rc *RobotClient) CurrentInputs(ctx context.Context) (referenceframe.FrameSystemInputs, error) {
+	panic("this feels wrong")
```

Comment:
@cheukt I'm not sure what to do here

Reply:
```
for _, name := range rc.ResourceNames() {
	input := referenceframe.NewZeroInputs(fs)
	
	res, err := rc.ResourceByName(name)
	if err != nil {
		return nil, err
	}
	inputEnabled, ok := res.(InputEnabled)
	if ok {
		pos, err := inputEnabled.CurrentInputs(ctx)
		if err != nil {
			return nil, err
		}
		input[name] = pos
	}
}
return input, nil
```
?

Reply:
Not sure if something bonus needs to be done for remotes

Reply:
Actually yeah that should work

Reply:
that would be what I'd expect as well